src/yall.mjs

```diff
@@ -36,24 +40,26 @@ export default function (options) {
       element.load();
     }
 
+    const classList = element.classList;
+
     // Lazy load CSS background images
-    if (element.classList.contains(lazyBackgroundClass)) {
-      element.classList.remove(lazyBackgroundClass);
-      element.classList.add(options.lazyBackgroundLoaded || "lazy-bg-loaded");
+    if (classList.contains(lazyBackgroundClass)) {
+      classList.remove(lazyBackgroundClass);
+      classList.add(options.lazyBackgroundLoaded || "lazy-bg-loaded");
     }
   };
 
   // Added because there was a number of patterns like this peppered throughout
   // the code. This just flips necessary data- attrs on an element
   const yallFlipDataAttrs = element => {
-    ["srcset", "src", "poster"].forEach(dataAttr => {
```

Comment:
Hm. Last I did this, it added some bytes. But, this approach means we're not instantiating an array every time we flip data attributes. This is probably more sound.